### PR TITLE
Remove non-alpha features from jobs.

### PR DESCRIPTION
**Pull request by rayandas**
This PR remove non-alpha features from job ci-kubernetes-e2e-gci-gce-alpha-features and pull-kubernetes-e2e-gce-alpha-features.
Partially fixes https://github.com/kubernetes/test-infra/issues/23642

Signed-off-by: Rayan Das <rayandas91@gmail.com>

### CODE REVIEW

**Comment by rayandas** [comment]
cc: @mmiranda96

**Comment by mmiranda96** [comment]
Thanks for your work @rayandas ! Let's run these jobs and see the result.
/test pull-kubernetes-e2e-gce-alpha-features
/test ci-kubernetes-e2e-gci-gce-alpha-features

**Comment by mmiranda96** [comment]
I think I can't trigger the CI job. Triggering just the PR job...
/test pull-kubernetes-e2e-gce-alpha-features

**Comment by rayandas** [comment]
@mmiranda96 I think we cant trigger pull-kubernetes-e2e-gce-alpha-features job also. It has to be one from the bot's message above.

**Comment by mmiranda96** [comment]
Yeah, I'm not sure why the job doesn't trigger. I'll ask about it on Slack.

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
FWIW I'm inclined to suggest we just merge this and watch the CI job to see if it breaks.  If it does, we can revert and take the more cautious route of duplicating the presubmit as a manually triggered job under a new name like pull-kubernetes-e2e-gce-alpha-features-canary

/hold
Please `/hold cancel` if you agree with this approach, or update the PR to take the more cautious approach first

**Comment by mmiranda96** [comment]
Considering that this PR only makes the test set smaller, I see very low risk.
/lgtm
/hold cancel

Thanks everyone!

**Pull request merged**
